scripts/reporting/ncanda_quality_control_script.py

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/env python
+##
+##  Copyright 2015 SRI International
+##  License: https://ncanda.sri.com/software-license.txt
+##
+##  $Revision: 2110 $
+##  $LastChangedBy: nicholsn $
+##  $LastChangedDate: 2015-08-07 09:10:29 -0700 (Fri, 07 Aug 2015) $
+##
+"""
+ncanda_quality_control_script
+======================
+This script checks the quality of the data for the NCANDA Project on REDCap.
+"""
+import os
+import sys
+import json
+import datetime
+
+import redcap
+import math
+import pandas as pd
+
+fields = ['study_id', 'redcap_event_name','exclude', 'visit_ignore',
+          'visit_date', 'dob', 'cnp_test_sessions_dob','saliva_missing',
+          'saliva_1_collected','saliva_1_date','saliva_2_collected','saliva_2_date',
+          'saliva_3_collected','saliva_3_date','saliva_4_collected',
+          'saliva_4_date','youthreport1_missing','youthreport1_date',
+          'youthreport1b_missing', 'youthreport1b_date','youthreport2_missing',
+          'youthreport2_date','youthreport2_yid2', 'youthreport1_yid2',
+          'parentreport_missing','parentreport_date','ssage_youth_missing',
+          'ssage_youth_date', 'lssaga1_youth_missing','lssaga1_youth_date',
+          'lssaga1_parent_missing','lssaga1_parent_date','bio_np_missing',
+          'bio_np_date','dd1000_missing','dd1000_date','dd100_missing',
+          'dd100_date','np_wrat4_missing','np_wrat4_wr_raw','np_gpeg_missing',
+          'np_gpeg_exclusion','np_gpeg_dh_time','np_gpeg_ndh_time',
+          'np_reyo_missing','np_reyo_copy_time','np_reyo_qc(completed)',
+          'np_atax_missing','np_atax_sht_trial1','np_wais4_missing',
+          'np_wais4_rawscore','np_wais4_rawscore_computed',
+          'np_wais4_rawscore_diff(correct)','pasat_missing','pasat_date',
+          'cnp_missing','cnp_test_sessions_dotest','stroop_missing',
+          'stroop_date','mrireport_missing','mrireport_date',
+          'mr_session_report_complete'];
+
+form_fields = [['youthreport1_missing','youthreport1_date'],
+          ['youthreport1b_missing', 'youthreport1b_date'],['youthreport2_missing',
```

Comment:
@abonil91 pls make sure that indentation is correct, here the lists should be properly aligned for readability. For example:

```
form_fields = [['youthreport1_missing','youthreport1_date'],
                      ['youthreport1b_missing', 'youthreport1b_date'],
                      ['youthreport2_missing', 'youthreport2_date'], 
...
``